report/src/bluetooth_presence_methods/proximity_detection.tex

```diff
@@ -0,0 +1,19 @@
+\section{Proximity detection}\label{sec:proximity_detection}
+Proximity detection computes distances using the Received Signal Strength Indicator (RSSI).
+Utilizing RSSI values, the proximity of devices, such as smartphones or wearables, to a central hub in the meeting room can be determined. This determination is based on the fact that as the signal strength diminishes, the distance between the device and the hub increases, providing a rudimentary approximation of the user's location.\cite{spachosBLEBeaconsIndoor2020}
+
+The principle of proximity detection is fundamentally rooted in the path-loss model, which correlates the received signal strength with the distance from the transmitter. This relationship can be formulated mathematically as follows:
+
+$$
+d = 10^{ \frac{(RSSI_{tx} - RSSI_{rx})}{10 \cdot n}}
+$$\cite{spachosBLEBeaconsIndoor2020}
+
+In this equation, \(d\) symbolizes the estimated distance, \(RSSI_{tx}\) stands for the transmitted signal strength, \(RSSI_{rx}\) denotes the received signal strength, and \(n\) is the path loss exponent. The path loss exponent, \(n\), varies depending on the environment. In open spaces, \(n\) is typically around 2, while in obstructed spaces, \(n\) can rise to at most 4.\cite{spachosBLEBeaconsIndoor2020, mathuranathanLogDistancePath2013}
```

Comment:
Rephrase such that it is clear that the value of n is a value that is chosen based on the environment